datafaker/generators.py

```diff
@@ -1587,6 +1589,23 @@ def get_generators(self, columns: list[Column], engine: Engine):
                 partition_def.nones,
                 {},
             )
+            query = self.query(
+                table=table,
+                columns=partition_def.included_numeric,
+                predicates=partition_def.predicates,
+                group_by_clause=partition_def.group_by_clause,
+                constants = partition_def.constants,
+                constant_clauses=partition_def.constant_clauses,
+                sample_count=self.SAMPLE_COUNT,
+            )
+            row_partitions_sampled[partition_def.index] = RowPartition(
+                query,
+                partition_def.included_numeric,
+                partition_def.included_choice,
+                partition_def.excluded,
+                partition_def.nones,
+                {},
+            )
             query = self.query(
```

Comment:
Maybe this should be renamed query_with_supression? so we're not reusing the same variable?

Reply:
OK, done!